### PR TITLE
Bootstrap-ify the usergroup_admin page

### PR DESCRIPTION
**Pull request by robertdown**

### CODE REVIEW

File: interface/themes/style_light.css [hunk 1]
```diff
@@ -80,12 +80,6 @@ a:hover {
   font-weight: bold;
 }
 
-.alert {
-  font-size: 70%;
-  color: #FF0000;
-  text-decoration: none;
-}
```
Comment (bradymiller) [inline interface/themes/style_light.css]: did we confirm this class is not being used elsewhere?

File: interface/themes/style_light.css [hunk 2]
```diff
@@ -1505,3 +1499,9 @@ table.dataTable#therapy_groups_list tbody td{
 .sidebar {
   background-color: #f8f8f8;
 }
+
+.alert-danger {
+  font-size: 100% !important;
+  background-color: #FF0000 !important;
+  color: #fff !important;
+}
```
Comment (bradymiller) [inline interface/themes/style_light.css]: Do the changes in the style also need to be completed in the other style?

File: interface/usergroup/usergroup_admin.php [hunk 3]
```diff
@@ -359,8 +359,8 @@
 ?>
 <html>
 <head>
-
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<title><?php xl('User / Group', 'e');?></title>
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/bootstrap-3-3-4/dist/css/bootstrap.css" type="text/css">
```
Comment (bradymiller) [inline interface/usergroup/usergroup_admin.php]: for "bootstrap light light", to ensure compatible when got to full standard assets headers in future, shouldn't we:
1. include the bootstrap after include the $css_header
2. include the bootstrap rtl style if applicable

File: interface/themes/style_light.css (hunk 1 above)
Comment (bradymiller) [inline interface/themes/style_light.css]: Also noted the alert class was still left on the users gui??

**Comment by bradymiller** [comment]
Tested well(looks really nice!). See my comments for minor issues to address. thanks! -brady

File: interface/themes/style_light.css (hunk 1 above)
Comment (robertdown) [inline interface/themes/style_light.css]: All styles should already have the alert class, I just wanted this one to look better.

File: interface/themes/style_light.css (hunk 2 above)
Comment (robertdown) [inline interface/themes/style_light.css]: Nope, this is a bootstrap class, I just overrode it in light because I wanted to give it a different look

**Comment by robertdown** [comment]
Going to merge this as I addressed the minor issues

File: interface/usergroup/usergroup_admin.php [hunk 4]
```diff
@@ -359,9 +359,12 @@
 ?>
 <html>
 <head>
-
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<title><?php xl('User / Group', 'e');?></title>
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/bootstrap-3-3-4/dist/css/bootstrap.css" type="text/css">
+<?php if ($GLOBALS['language_direction'] == 'rtl'): ?>
```
Comment (bradymiller) [inline interface/usergroup/usergroup_admin.php]: This should be:
``````
$_SESSION['language_direction']

Comment (bradymiller) [inline interface/usergroup/usergroup_admin.php]: i'll bring this fix in; also have a quick style fix for the cancel button in procedure form to bring in, so will bring it with that.

**Comment by robertdown** [comment]
Dang it, sorry.... hate bringing in bad code. My bad!

**Comment by bradymiller** [comment]
no prob, fix is already in the codebase :)

**Pull request merged**